week-6-benchmarking/api/app.py

```diff
@@ -0,0 +1,41 @@
+import torch
+from model import RegressionModel
+from transformers import BertTokenizer
+from flask import Flask, request, Response
+import json
+
+
+device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
+
+print(f">>> using device: {device}")
+
+model = RegressionModel()
+model.load_state_dict(torch.load("./weights/bert-base-uncased-regression-weights.pth"))
+model.to(device)
+tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
+model.eval()
```

Comment:
would be nice to wrap this block into a function

Reply:
done